interface/HH_BTag.h

```diff
@@ -0,0 +1,36 @@
+#include <vector>
+#include <string>
+#include "PhysicsTools/TensorFlow/interface/TensorFlow.h"
+
+namespace hh_btag{
+
+namespace InputVars{
+    enum vars { jet_valid = 0, jet_pt = 1, jet_eta = 2, rel_jet_M_pt = 3, rel_jet_E_pt = 4, jet_htt_deta = 5,
+                jet_deepFlavour = 6, jet_htt_dphi = 7, sample_year= 8, channelId = 9, htt_pt = 10, htt_eta = 11,
+                htt_met_dphi = 12, rel_met_pt_htt_pt = 13, htt_scalar_pt = 14
+    };
+ }
+
+class HH_BTag {
+public:
+    HH_BTag(const std::string& model);
+    ~HH_BTag();
+
+    std::vector<float> GetScore(const std::vector<float>& jet_valid, const std::vector<float>& jet_pt,
+                                const std::vector<float>& jet_eta, const std::vector<float>& rel_jet_M_pt,
+                                const std::vector<float>& rel_jet_E_pt, const std::vector<float>& jet_htt_deta,
+                                const std::vector<float>& jet_deepFlavour, const std::vector<float>& jet_htt_dphi,
+                                const float& sample_year, const float& channelId, const float& htt_pt,
```

Comment:
- all `const float&` -> `float`
- parity of the event should be added as an argument.